### PR TITLE
feat: isolate email messages into a configuration file

### PR DESCRIPTION
**Pull request by cs12b006**
- Abstracts out literals
- Handlebar app integration

### CODE REVIEW

File: webpack.config.ts [hunk 1]
```diff
@@ -17,10 +17,18 @@ const config: webpack.Configuration = {
         use: "ts-loader",
         exclude: /node_modules/,
       },
+      {
+        enforce: 'pre',
+        test: /\.html$/,
+        loader: 'raw-loader',
+      },
```
Comment (favonia) [inline webpack.config.ts:24]: Do we need this in this PR?

File: package.json [hunk 2]
```diff
@@ -35,6 +38,8 @@
   },
   "dependencies": {
     "@types/google-apps-script": "^1.0.55",
+    "ci": "^2.2.0",
```
Comment (favonia) [inline package.json:41]: Why do we need this?

File: webpack.config.ts (hunk 1 above)
Comment (cs12b006) [reply webpack.config.ts:24]: Yes, this can go to another PR.

File: package.json (hunk 2 above)
Comment (cs12b006) [reply package.json:41]: I had this to smoothly resolve package versions in different branches. I suppose I can move it to dev dependencies.

File: literals/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## Literals
+The directory contains all the literals that are used in emails sent out. The `LiteralType` contains a type with `subject`, `body` and `greeting` (optional) as needed for all the emails. All the `literal` Objects defined to handle the different possible responses are as listed below.
```
Comment (favonia) [inline literals/README.md:2]: I have trouble understanding this paragraph, especially the last sentence. Is there a typo? What are "`literal` Objects"?

**Comment by favonia** [comment]
@cs12b006 Did you test the system with literal to make sure that the email messages produced by the new system are essentially the same?

File: literals/README.md (hunk 3 above)
Comment (cs12b006) [reply literals/README.md:2]: Let me change it to "`literal` Object is defined to handle as listed below."

**Comment by cs12b006** [comment]
> @cs12b006 Did you test the system with literal to make sure that the email messages produced by the new system are essentially the same?

I have test cases to cover all cases in another branch (that validate message bodies). I will do a dry run for all the possible cases with the latest PR code and update here in 10 min.

File: literals/README.md (hunk 3 above)
Comment (favonia) [reply literals/README.md:2]: Why is "Object" capitalized? Actually, could you commit the supposed change?

**Comment by cs12b006** [comment]
> @cs12b006 Did you test the system with literal to make sure that the email messages produced by the new system are essentially the same?

The code at present on this PR is giving the expected responses for all the email cases. Also, I fixed some part in the `README.md` for literals.

**Comment by favonia** [comment]
Alright let me merge this PR first.

**Pull request merged**